src/index.html

```diff
@@ -17,16 +17,25 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="form-field">
+      <form data-qa="big">
```

Comment:
The `data-qa` attribute for the form should be `big-search-form` instead of `big` according to the task requirements.